config/config.json

```diff
@@ -32,7 +32,7 @@
         "MaxUsersPerTeam": 50,
         "EnableTeamCreation": true,
         "EnableUserCreation": true,
-        "EnableOpenServer": false,
```

Comment:
This is a mistake right? I don't think we want to change the default people use for this setting